src/chat-bubble/chat-bubble-base.ts

```diff
@@ -0,0 +1,68 @@
+import { html, LitElement, nothing } from "lit";
+import { property } from "lit/decorators.js";
+import { logger } from "../utils";
+import { AUTHORS, BaseParts, BaseSlots } from "./constants";
+
+export class ChatBubbleBase extends LitElement {
+  @property({ type: String, reflect: true })
+  public author!: (typeof AUTHORS)[number];
+
+  @property({ type: String, reflect: true })
+  public timestamp!: string;
+
+  @property({ type: Boolean, reflect: true, attribute: "fully-rounded" })
+  public fullyRounded: boolean = false;
```

Comment:
I think it’s better to align with the naming conventions used in the design system. In the design system, this property is referred to as first-message or at least first, correct?